squiggy/models/asset.py

```diff
@@ -56,7 +57,7 @@ class Asset(Base):
         backref='assets',
     )
     users = db.relationship(
-        'User',
+        User.__name__,
```

Comment:
Prior to this change, I was hitting: "sqlalchemy.InvalidRequestError: When initializing mapper mapped class Asset->assets, expression 'User' failed to locate a name ('User')"